pom.xml

```diff
@@ -51,7 +51,7 @@
         <i2p-eddsa.version>0.3.0</i2p-eddsa.version>
         <!-- this must be the same value in Dockerfile
             via `gauge install java -version <x.y.z>`  -->
-        <gauge-java.version>0.7.7</gauge-java.version>
+        <gauge-java.version>0.7.15</gauge-java.version>
 
         <!-- plugins -->
```

Comment:
what about upgrading `web3j-maven-plugin` and `web3j.version`?

Reply:
I'm still in the process of upgrading. There is some API that changed as well on the side of web3j, so some code needs updating.
I didn't flag this PR as draft. I'll do it right now.

Reply:
on a separate PR